decanter_ai_sdk/decanter_client.py

```diff
@@ -0,0 +1,623 @@
+from io import StringIO
```

Comment:
I think the name can still be client. So we have `client` and `non_blocking_client`